api/v1alpha4/types.go

```diff
@@ -133,7 +133,7 @@ type RouteTable struct {
 	Name string `json:"name,omitempty"`
 }
 
-// NatGateway defines an Azure Nat Gateway.
+// NatGateway defines an Azure NAT gateway.
```

Comment:
let's not change v1alpha4 types, even if it's just code comments (those are actually used as CRD descriptions)